exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/ColumnDefinitionTests.kt

```diff
@@ -0,0 +1,195 @@
+        val tester = object : Table("tester") {
+            val amount = integer("amount").withDefinition("COMMENT", stringLiteral(comment))
+        }
+
+        val columnCommentSupportedDB = TestDB.ALL_H2.toSet() + TestDB.ALL_MYSQL_MARIADB + TestDB.SQLITE
+
+        withTables(excludeSettings = TestDB.ALL - columnCommentSupportedDB, tester) { testDb ->
+            assertTrue { SchemaUtils.statementsRequiredToActualizeScheme(tester).isEmpty() }
+
+            tester.insert { it[amount] = 9 }
+            assertEquals(9, tester.selectAll().single()[tester.amount])
+
+            val tableName = tester.nameInDatabaseCase()
+            val showStatement = when (testDb) {
+                in TestDB.ALL_MYSQL_MARIADB -> "SHOW FULL COLUMNS FROM $tableName"
+                TestDB.SQLITE -> "PRAGMA table_info($tableName)"
+                else -> "SELECT REMARKS FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = '$tableName'"
+            }
+            val resultLabel = when (testDb) {
+                in TestDB.ALL_MYSQL_MARIADB -> "Comment"
+                TestDB.SQLITE -> "type"
+                else -> "REMARKS"
+            }
+
+            val result = exec(showStatement) { rs ->
+                rs.next()
+                rs.getString(resultLabel)
+            }
+            assertNotNull(result)
+            assertContains(result, comment)
+        }
+    }
+
+    @Test
+    fun testColumnDataMasking() {
+        val tester = object : Table("tester") {
+            val email = varchar("email", 128).uniqueIndex().withDefinition("MASKED WITH (FUNCTION = 'email()')")
+        }
+
+        withTables(TestDB.ALL - TestDB.SQLSERVER, tester) {
+            assertTrue { SchemaUtils.statementsRequiredToActualizeScheme(tester).isEmpty() }
+
+            val testEmail = "mysecretemail123@gmail.com"
+            tester.insert {
+                it[email] = testEmail
+            }
+
+            // create a new user with limited permissions
+            exec("CREATE USER MaskingTestUser WITHOUT LOGIN;")
+            exec("GRANT SELECT ON ${tester.nameInDatabaseCase()} TO MaskingTestUser;")
+            exec("EXECUTE AS USER = 'MaskingTestUser';", explicitStatementType = StatementType.OTHER)
+
+            // Email function obfuscates data of all length to form 'aXXX@XXXX.com', where 'a' is original first letter
+            val maskedEmail = "${testEmail.first()}XXX@XXXX.com"
+            val maskedResult = tester.selectAll().single()[tester.email]
+            assertEquals(maskedEmail, maskedResult)
+
+            exec("REVERT;")
+            exec("DROP USER MaskingTestUser;")
+        }
+    }
+
+    @Test
+    fun testColumnDefaultOnNull() {
+        val itemA = "Item A"
+        val tester = object : Table("tester") {
+            val amount = integer("amount")
+            var item: Column<String> = varchar("item", 32)
+
+            fun initItemColumn() {
+                (columns as MutableList<Column<*>>).remove(item)
```

Comment:
I was surprised, but it's really not possible to do with `replaceColumn()`, because `varchar()` internally tries to register column a second time... probably we could add method `removeColumn`, or something like `item.replaceColumn { varchar("tester")... }` for better experience with column manipulation from outside of the library. But it's just thoughts, it's not the problem of PR itself.

Reply:
Anyway, as an alternative:

```kotlin
withDb(TestDB.ALL_H2_V2 + TestDB.ORACLE) { testDb ->
  val tester = object : Table("tester") {
      val amount = integer("amount")
      var item: Column<String> = if (testDb in TestDB.ALL_H2) {
          varchar("item", 32).default(itemA).withDefinition("DEFAULT ON NULL")
      } else {
          varchar("item", 32).withDefinition("DEFAULT ON NULL", stringLiteral(itemA))
      }
  }
  ...
}
```

It could be just a bit more readable, in the rest the same.

Reply:
I considered this and hesitated because I wondered if it read weird. Since I'm not the only one, I'll switch to this more simple way.